readme.md

```diff
@@ -0,0 +1,21 @@
+## Local Development
+
+---
+
+### Build docker image and up all entities
```

Comment:
Just few comments:
- which version of python do we use?
- which dependencies we need (pip install ...)?
- do we use venv or other environment system? 

These questions are often asked by beginners, and not only. They are needed to synchronize the work of the team so that there are no ambiguities. These are not strict rules, but I advise you to pay attention to them. Sooner or later you will have to work in a large team